doc/nrf/ug_thread_certification.rst

```diff
@@ -0,0 +1,109 @@
+
+
+Certification by inheritance without modifications to binaries
+==============================================================
+
+If you are developing a Thread product with nRF Connect SDK, you can apply for certifying this product by inheritance as an official Thread certified product.
+This is possible for products that are based on systems from Nordic (both stack and device) that were previously certified by Thread Group, as long as you have not modified the initial functionality that was tested by Thread Group for certification.
+
+This scenario offers the most simplified and potentially the shortest certification process at the lowest cost.
+It is available to `Thread Group members`_ at Implementer tier or higher.
+
+Check :ref:`nrfxlib:ot_libs` for the latest available OpenThread certified libraries with nRF Connect SDK and their usage.
+
+Certification by inheritance with modifications to binaries
+===========================================================
+
+If your solution uses the same version of the OpenThread stack as the one used in the precompiled, certified binaries, but you made modifications within the operating parameters defined by Thread Group, you need to contact Thread Group to check if certification by inheritance is still possible.
+You have to officially list all changes with a detailed explanation in your application.
+Based on this list, Thread Group can accept or reject your application for certification by inheritance.
+
+Certification at ATL
+====================
+
+If your solution modifies the stack's certified functionality or if Thread Group rejects your application for certification by inheritance, you need to undergo the full certification procedure at an ATL.
+Only Contributor and Sponsor tier members of Thread Group can apply for certification by an ATL.
+
+Thread Test Harness integration
+*******************************
+
+To certify a Thread product, all certification test cases must be run on the Thread Test Harness software.
+
+A detailed description of how to assemble and configure a Thread Test Bed and run the Thread Test Harness can be found on the `Thread Group's Confluence wiki page`_.
+
+.. note::
+   The following procedure references the nRF52840 Development Kit.
+   The analogical procedure may be used to run the certification using other development kits.
+
+Complete the following steps to run the certification tests:
+
+
+#. Build the certification image.
+
+   The :ref:`ot_cli_sample` sample is used as a base, modifying it with an overlay file.
+
+   .. code-block::
+
+         cd ncs/nrf/samples/openthread/cli/
+         west build -b nrf52840dk_nrf52840 -- -DCONF_FILE="prj.conf harness/overlay-cert.conf"
+
+   .. note::
+      The overlay file selects the precompiled OpenThread libraries by default.
+
+#. Prepare Thread Test Harness.
+
+   a. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness/nrf52840-ncs.py` file into :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI`.
```

Comment:
Does this path need to start with ``C:\GRL`` or is it optional?  @edmont

Reply:
It's the default installation path for the Thread Test Harness. I imagine some other path can be selected during the installation process.